Description: Fixes the sqlalchemy requirement
 Upstream wrongly restricts SQLAlchemy dependency, while 0.8 is perfectly
 fine with Neutron.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-12-02

--- neutron-2013.2.orig/requirements.txt
+++ neutron-2013.2/requirements.txt
@@ -17,7 +17,7 @@ Jinja2
 kombu>=2.4.8
 netaddr
 python-neutronclient>=2.3.0,<3
-SQLAlchemy>=0.7.8,<=0.7.99
+SQLAlchemy>=0.7.8
 WebOb>=1.2.3,<1.3
 python-keystoneclient>=0.3.2
 alembic>=0.4.1
